### ISSUE TITLE
Segregation of standard output and standard error streams at Web UI level

### ISSUE THREAD

**Issue by ghansham** [dagu-org/dagu#687]
Dear Sir 

Kindly create two separate links for viewing standard output and standard error streams of dag processes. We can also make the error code returned as a link and redirect user ro stderr stream and user can go to stdout lmstream from existing link.
Another improvement, we suggest is to give provision to the user to utilize params in the dag to form the log filenames for stdout and err if specified.

Regards
Ghansham

**Comment by kriyanshii** [comment]
can you assign this to me?

**Comment by yohamta** [comment]
Yes, sure. Please ask anything if there's any issues or questions! Thanks a lot!

**Comment by kriyanshii** [comment]
Hey, already I have got this working.
But if you want to do this nevermind

**Comment by yottahmd** [comment]
Oh hey, @kriyanshii,  are you working off the next branch? If so, feel free to take it on. Thank you!

**Comment by kriyanshii** [comment]
Yeah working it on the next branch!😁

**Comment by yottahmd** [comment]
Great, thanks!

**Comment by ghansham** [comment]
With 1.17alpha, I think this is close able.

**Comment by yottahmd** [comment]
That's true, thank you!

**Comment by ghansham** [comment]
Looks like we may have to keep this feature optional at dag level. It may need changes both on server and client side.
At dag level, can we introduce a field:
```yaml
logging: single #default for backward compatibility with 1.16.x
or
logging: split
```
It may be linked to #312.
